compiler/quilt/tools/command.py

```diff
@@ -1279,28 +1279,31 @@ def reset_password(team, username):
             ), data=json.dumps({'username':username})
     )
 
-def _load(package):
+def _load(package, hash=None):
     info = parse_package_extended(package)
```

Comment:
Instead of using parse_package_extended then raising an exception if tags, etc are specified, just use parse_package. It might be better to move the checks for correct extended syntax to parse_package.

Reply:
I looked carefully into this and it involves more surgery and risk than necessary. In particular I'd have to import the `PackageInfo` named tuple from `util.py` and use it to wrap the return values of `parse_package`, which is silly because `PackageInfo` is the native return type of `parse_package_extended`.